package.json

```diff
@@ -124,10 +124,10 @@
 		"vitest": "^2.0.3",
 		"webpack": "^5.93.0",
 		"webpack-cli": "^5.1.4",
-		"xo": "^0.58.0"
+		"xo": "^0.59.0"
 	},
 	"engines": {
-		"node": ">= 20",
+		"node": "~21",
```

Comment:
- 21 because of https://github.com/xojs/xo/issues/598
- ~21 because of https://github.com/npm/cli/issues/7657